src/kinvey.js

```diff
@@ -184,6 +185,8 @@ Kinvey.AuthorizationGrant = AuthorizationGrant;
 Kinvey.CustomEndpoint = CustomEndpoint;
 Kinvey.DataStore = DataStore;
 Kinvey.DataStoreType = DataStoreType;
+Kinvey.Stream = Stream;
+Kinvey.LiveService = LiveServiceFacade;
```

Comment:
As per my latest changes (which I haven't pushed yet, but hope to do so before standup today), I've attached `StreamACL` to `Stream` and I will move `Stream` to `LiveServiceFacade`. Think this should work well indeed.